druid-0.23-release-notes.md

```diff
@@ -79,25 +238,34 @@ Users can now cancel SQL queries just like native queries can be cancelled. A ne
 
 ## Other changes
 
+- Druid now processes lookup load failures more quickly (#12397)
+- `BalanceSegments#balanceServers` now exits early when there is no balancing work to do (#11768)
+- `DimensionHandler` now allows you to define a `DimensionSpec` appropriate for the type of dimension to handle (#11873)
+- Added an interface for external schema providers to Druid SQL (#12043)
+- Added support for a SQL INSERT planner (#11959)
+
 # Security fixes
 
 ## Support for access control on setting query contexts
 
 Today, any context params are allowed to users. This can cause 1) a bad UX if the context param is not matured yet or 2) even query failure or system fault in the worst case if a sensitive param is abused, ex) maxSubqueryRows. Druid now has an ability to limit context params per user role. That means, a query will fail if you have a context param set in the query that is not allowed to you. 
 
-The context param authorization can be enabled using druid.auth.authorizeQueryContextParams. This is disabled by default to avoid any hassle when performing an upgrade.
+The context parameter authorization can be enabled using druid.`auth.authorizeQueryContextParam`s. This is disabled by default to avoid any hassle when performing an upgrade.
 
-[12396](https://github.com/apache/druid/pull/12396)
+(#12396)
 
-## 
+## Other security improvements
 
-# Performance improvements
+This release includes several additional improvements for security:
+- You can now optionally enable auhorization on Druid system tables (#11720)
+
+## Performance improvements
 
 ### General performance
 
 ### Ingestion
 
-- More accurate memory estimations while building an on-heap incremental index. Rather than using the max possible size of an aggregated row, Druid can now use (based on a task context flag) a closer estimate of the actual heap footprint of an aggregated row. This enables the indexer to fit more rows in memory before doing an intermediate persist. [12073](https://github.com/apache/druid/pull/12073)
+- More accurate memory estimations while building an on-heap incremental index. Rather than using the max possible size of an aggregated row, Druid can now use (based on a task context flag) a closer estimate of the actual heap footprint of an aggregated row. This enables the indexer to fit more rows in memory before doing an intermediate persist. (#12073)
```

Comment:
```suggestion
- More accurate memory estimations while building an on-heap incremental index. Rather than using the maximum possible aggregated row size, Druid can now use (based on a task context flag) a closer estimate of the actual heap footprint of an aggregated row. This enables the indexer to fit more rows in memory before performing an intermediate persist. (#12073)
```